index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
+<h2 hidden>Game over BISH</h2>
```

Comment:
Bit too agressive innit?
```suggestion
<h2 hidden>Game over Frtiend</h2>
```